code/src/App.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './Album'
+import { Header } from './Header'
 
 console.log(data)
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
-    </div>
+      <Header />
+      <div className="album-container">
+      {data.albums.items.map(item => {
+        return <Album key={item.id}
+          cover={item.images[1].url}
+          artistURL={item.artists[0].external_urls.spotify}
+          albumURL={item.external_urls.spotify}
+          name={item.name} artist={item.artists[0].name} />
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of artistURL and artist go for artists={album.artists}